src/components/ProcessedReleaseChangeDescription.tsx

```diff
@@ -68,10 +68,20 @@ const RemarkOl = (props: unknown) => (
 const RemarkLi = (props: ListItemProps) => <ListItem {...props} />
 
 const RemarkPre = (props: unknown) => (
-  <Code as="pre" display="block" mb="4" p="3" overflowX="auto" {...props} />
+  <Code
+    as="pre"
+    display="block"
+    bgColor="primaryBg"
```

Comment:
The design didn't include any example with code blocks, so I did my best and set the bg color of those blocks to `primaryBg` to have some contrast with `secondaryBg`. Imho, it looked better before (left side) than now (right side):
<img width="835" alt="CleanShot 2022-01-30 at 20 41 55@2x" src="https://user-images.githubusercontent.com/2677072/151714964-69d3eae8-6735-41bc-b8a7-775b6ed6c3fc.png">

I think we should leave the comparator page background with the lightest/darkest gray, to put the code block with a little bit darker/lighter background as before. What do you think?

Reply:
I actually like the codeblocks in this PR better!

The previous design had too much emphasis on the code blocks, now the codeblocks fit in better with the rest of the page.

There's a lot of custom colors coming from a syntax highlight extension/plugin/program. Can we control any of those colors?

It will be very tricky to get the colors just right on our light/dark codeblocks.

Reply:
Ok, fair enough.

We can control those colors indeed. I'll investigate how to override them for the dark mode in the proper ticket!